### DISCUSSION TITLE
Add a function to get all hyperlink in a worksheet

### DISCUSSION THREAD

**Discussion by undeflife** [qax-os/excelize#1607] [Ideas]
currently, we can only  get hyperlink cell by cell which is inefficient
```go
rows, _ := e.GetRows(sheetName)
for rowNumber, row := range rows{
    for cellNumber, cell := range cols {
        cellRef := toCellRef(rowNumber, cellNumber+1)
        if ok, link, _ := e.GetCellHyperLink(sheetName, cellRef); ok {
        // do something to the link 
        }
    }
}
```
 provide a function to get all external hyperlinks will enable a more flexible way to deal with hyperlinks 

```go
rows, _ := e.GetRows(sheetName)
links, _ := e.Hyperlinks(sheetName)
// k is a cellRef like 'A1' v
for k, v := range links {
  rowNumber, cellNumber := toCellPosition(k)
 fmt.Printf("cell %s : value is %s and has link to %s\n ", k, rows[rowNumber][cellNumber-1], v)
}
```

please comments on this and I am glad to open a pull request to implement it.

**Comment by xuri** [comment]
Thanks for your feedback. I have added new function `GetHyperLinkCells` for this. Please upgrade to the master branch code by `go get -u github.com/xuri/excelize/v2@master`, and this feature will be released on the next version.
